src/screens/Learn/Query/Query.jsx

```diff
@@ -126,8 +125,11 @@ const Query = () => {
     if (!vocabs) {
       return;
     }
+    let timer1 = setTimeout(() => setCurrVocab(vocabs[0]), 260);
 
-    setCurrVocab(vocabs[0]);
+    return () => {
```

Comment:
If you go with this timeout logic, you have to disable the `correct` and `wrong` button for that time. Otherwise we will submit the same vocab a lot of time and the status bar increases if we spam the correct button and the card has no time to turn around.
![image](https://user-images.githubusercontent.com/60048565/120900715-a3964500-c636-11eb-9c1e-25ceb91045a6.png)

Reply:
done